finagle-toggle/src/main/scala/com/twitter/finagle/toggle/JsonToggleMap.scala

```diff
@@ -126,13 +126,13 @@ object JsonToggleMap {
       }
 
       val metadata: immutable.Seq[Toggle.Metadata] =
-        toggles.map { jsonToggle =>
+        toggles.map(jsonToggle => {
```

Comment:
why did we have to switch this to use curlies inside instead of outside?

Reply:
I could have sworn I had some deprecation warnings for this,  but I can't reproduce that anymore, so it can be changed back.